lightning/src/chain/keysinterface.rs

```diff
@@ -413,7 +427,17 @@ impl InMemoryChannelKeys {
 		}
 	}
 
-	fn remote_pubkeys<'a>(&'a self) -> &'a ChannelPublicKeys { self.remote_channel_pubkeys.as_ref().unwrap() }
+	/// Remote pubkeys
+	/// Will panic if the remote pubkeys were not set with on_accept
+	pub fn remote_pubkeys(&self) -> &ChannelPublicKeys { &self.accepted_channel_data.as_ref().unwrap().remote_channel_pubkeys }
+
+	/// Remote to-self delay, constraining local commitment txs
+	/// Will panic if the remote pubkeys were not set with on_accept
+	pub fn remote_to_self_delay(&self) -> u16 { self.accepted_channel_data.as_ref().unwrap().remote_to_self_delay }
+
+	/// Local to-self delay, constraining remote commitment txs
+	/// Will panic if the remote pubkeys were not set with on_accept
```

Comment:
"Will panic if accepted_channel_state isn't set", not only remote pubkeys.

Reply:
reworded in three places